hip_data_tools/aws/aws.py

```diff
@@ -1,15 +1,42 @@
 import boto3 as boto
```

Comment:
should the name of this module be more descriptive? Might be a bit confusing just to call it aws

Reply:
it handles the common aws stuff i.e. the aws connection manager... could call it aws/common.py